<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CategorySelectorDialog</class>
 <widget class="QDialog" name="CategorySelectorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>310</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>310</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>310</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Wheel of Jeopardy!</string>
  </property>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>16</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>83</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>461</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="textLabel">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="categoryLabel">
          <property name="text">
           <string>Category:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="categoryComboBox">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>29</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>110</width>
            <height>29</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>Category 1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Category 2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Category 3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Category 4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Category 5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Category 6</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="okButton">
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
